sources/core/Stride.Core.Mathematics/Quaternion.cs

```diff
@@ -677,6 +695,24 @@ public static Quaternion Lerp(Quaternion start, Quaternion end, float amount)
             Lerp(ref start, ref end, amount, out result);
             return result;
         }
+        
+        /// <summary>
+        /// Returns a rotation whose facing direction points towards <paramref name="forward"/>
+        /// and whose up direction points as close as possible to <paramref name="up"/>.
+        /// </summary>
+        public static Quaternion LookRotation(in Vector3 forward, in Vector3 up)
```

Comment:
To a novice like myself, the concept of `forward` in this context is a bit hard to grasp. Could this have an overload to do the simple `Vector3.Normalize(target - source)` but takes a `source` and `target` parameter instead of `forward`?